nautobot/docs/installation/tables/v2-database-behavior-changes.yaml

```
@@ -50,6 +62,12 @@
 - Model: "VirtualChassis"
   Field: "name"
   Changes: "Now must be globally unique"
+- Model: "VirtualMachine"
+  Field: "primary_ip4"
+  Changes: "No longer one-to-one, now one-to-many (`ForeignKey`). Remote side is now a list."
+- Model: "VirtualMachine"
+  Field: "primary_ip6"
+  Changes: "No longer one-to-one, now one-to-many (`ForeignKey`). Remote side is now a list."
```

```suggestion
  Changes: "No longer one-to-one, now one-to-many (`ForeignKey`)"
```
`VirtualMachine.primary_ip6` uses `related_name="+"`, so no reverse relationship is created

---

```suggestion
  Changes: "Changed from `OneToOneField` to `ForeignKey`, making it possible for multiple virtual machines to share the same `primary_ip6`"
```